ddtrace/ext/priority.py

```diff
@@ -13,12 +13,13 @@
 # Indicate to keep the trace
 span.context.sampling_priority = USER_KEEP
 """
+from ddtrace.constants import AUTO_KEEP
```

Comment:
we also want to mark this module as deprecated:

https://github.com/DataDog/dd-trace-py/blob/388e4f577350d62687f60613b0a4f6aeb65c59a1/ddtrace/util.py#L11-L15

Reply:
👍